techniques/aria/ARIA23.html

```diff
@@ -40,7 +40,7 @@ <h3>Updating the contents of a chat conversation</h3>
 			</section>
 			<section class="example">
 				<h3>Updating the log of a server</h3>
-				<p>An application log records time-stamped activities. The log is exposed in the app as a view, with the region marked with the role of <code>log</code> so that the new additions are announced by the ATs. (The default value for the <code>aria-relevant</code> attribute is "additions", so the removal of the old top entries due to log size limitations will not be announced.)  When each new log entry is added, a screen reader announces it.</p>
+				<p>An application log records time-stamped activities. The log is exposed in the app as a view, with the region marked with <code>role="log"</code> so that the new additions are announced by the ATs. (The default value for the <code>aria-relevant</code> attribute is "additions", so the removal of the old top entries due to log size limitations will not be announced.)  When each new log entry is added, a screen reader announces it.</p>
```

Comment:
```suggestion
				<p>An application log records time-stamped activities. The log is exposed in the app as a view, with the region marked with <code>role="log"</code> so that the new additions are announced by the ATs. (The default value for the <code>aria-relevant</code> attribute is <code>"additions"</code>, so the removal of the old top entries due to log size limitations will not be announced.)  When each new log entry is added, a screen reader announces it.</p>
```

Maybe a `<code>` for "additions" too, since it’s a string literal?